src/components/Shared/DatePicker/index.js

```diff
@@ -0,0 +1,80 @@
+import Flatpickr from 'react-flatpickr';
+import moment from 'moment';
+import React, { Component } from 'react';
+
+import 'flatpickr/dist/themes/material_blue.css';
+import 'components/Shared/DatePicker/DatePicker.css';
+/**
+  Passes an estimated openDate via the "onChange" prop.
+  Unless the "Indefinite Closure" checkbox is ticked, then the crossing will not have an estimated openDate.
+**/
+export default class DatePicker extends Component {
+  stringToDate(dateString) {
+    return dateString ? moment(dateString).toDate() : null;
+  }
+
+  dateToString(date) {
+    return moment(date).format('YYYY-MM-DD');
+  }
+
+  render() {
+    const { openDate, indefiniteClosure } = this.props;
+    const flatpickrDate = this.stringToDate(openDate)
+
+    return (
+      <div className="duration-container">
+        <table className='duration-table'>
+          <tbody>
+            <tr>
+              <td>
+                Estimated Open Date:
```

Comment:
Nitpick, but I like the term reopen date better here. Perfectly open to keeping it as "open" instead, just figured I'd make sure my opinion is noted.

Reply:
Makes Sense. "Open" means something different in the context of our app. I'll switch it.